pkg/ingestor/parser/deps_dev/deps_dev.go

```diff
@@ -182,5 +138,16 @@ func (d *depsDevParser) GetIdentities(ctx context.Context) []common.TrustInforma
 }
 
 func (d *depsDevParser) GetIdentifiers(ctx context.Context) (*common.IdentifierStrings, error) {
-	return nil, fmt.Errorf("not yet implemented")
+	idstrings := &common.IdentifierStrings{}
+	for _, depComp := range d.packComponent.DepPackages {
```

Comment:
This does cause deps.dev to spin for a long time. Do we want this for the beta release?

Reply:
Future implementation would be that people would connect to the public instance to get this data and not keep going down the chain?

Reply:
deps.dev can handle pretty big QPS from my knowledge, however, i think based off https://github.com/google/deps.dev/issues/12, it seems like we have all the information we need to construct all that is needed... so if we implement a cache check in the deps.dev collector it should not result in a big hit rate (like for the sourceinput one but we can make it more generic if we have all the data available).

Reply:
Based on discussion, commented out the csub identifiers from deps.dev parser until the issue https://github.com/guacsec/guac/issues/769 is addressed.